.gitattributes

```diff
@@ -1,6 +1,5 @@
 _static/vid/**/* filter=lfs diff=lfs merge=lfs -text
```

Comment:
Not sure what/why the changes are here or if this might cause any problems.

Reply:
This file get changes when I tracked the images from git lfs. I have learned this thing from my previous issues. If there is something wrong Please make me correct